### PR TITLE
resolve double UTF-8 encode.

### PR DESCRIPTION
**Pull request by whtiehack**
https://github.com/github-tools/github/issues/463

why:
https://github.com/github-tools/github/blob/24223f7a51ba935f1c86ffd552dac20e8f3e359d/lib/Repository.js#L252

double utf-8 encode:
https://github.com/whtiehack/hello-world/pull/11/files

resolved:
https://github.com/whtiehack/hello-world/pull/12/files

### CODE REVIEW

**Comment by M3kH** [comment]
Hi @whtiehack thanks for the PR, I will check, merge and deploy; looks good :-)

**Comment by whtiehack** [comment]
ok :),thanks for you prbot tool.

File: src/PullRequest.js [hunk 1]
```diff
@@ -40,6 +40,9 @@ export default class PullRequest {
         this.commitMessage = `🤖 ${commitMessage || 'Anonymous Commit'}`;
         this.titlePullRequest = titlePullRequest;
         this.descriptionPullRequest = descriptionPullRequest;
+        for(let file of files){
+            file.content = Buffer.from(file.content)
+        }
         this.files = files;
```
Comment (M3kH) [inline src/PullRequest.js]:
```suggestion
        this.files = files.map( ({ path, content }) => ({ path, content: Buffer.from(content) }) );
```

**Comment by whtiehack** [comment]
nice.

**Comment by M3kH** [comment]
Thanks @whtiehack

**Pull request merged**